src/components/icon/icon.svelte

```diff
@@ -3,13 +3,23 @@
   import { writable } from 'svelte/store'
 
   let lastIconBasePath = '/icons'
-  let iconBasePath = writable(lastIconBasePath)
+  let iconBasePath = writable<{
+    [key: string]: string
+  }>({
+    '': lastIconBasePath
```

Comment:
Any way we could use something like `default` instead of an empty string?
```suggestion
    DEFAULT: lastIconBasePath
```

Reply:
We could do something like:

```
const defaultSymbol = new Symbol()

{
    [defaultSymbol]: lastIconBasePath
}
```